analysis_options.yaml

```diff
@@ -1,5 +1,10 @@
 include: package:pedantic/analysis_options.yaml
 
+analyzer:
+  errors:
+    always_declare_return_types: ignore # 318
```

Comment:
Are these the number of violations?

Reply:
Yup. Should have been more clear. Sorry!